### PR TITLE
update routings after modifying dedicated turning lanes policy

### LINKED ISSUE #1235: dedicated turning lane policy is only applied superficially
I activated dedicated turning lane policy in game and this happened:
image
The lane arrows are pointing to the right direction, but the routing manager doesn't seem to have been updated.

- ~~I tried force updating the segment using Network detective (`NetManger.UpdateNode/Segment()`) and even that didn't help.~~ EDIT: Update does work, it just takes a while to take effect.
- If I restart the game it works fine. 
- if I select the segment and press delete to reset lane arrows, that will also fix the problem (even though I had not forced any lane arrows).

### PR DESCRIPTION
**Pull request by kianzarrin**
fixes #1235 (see issue)

Tested and works in game. after deleted all cars new paths will obey dedicated lane arrows.
it is also pretty fast in 655k city (a few seconds)

### CODE REVIEW

File: TLM/TLM/Util/PlaceIntersectionUtil.cs [hunk 1]
```diff
@@ -54,9 +54,9 @@ public static class PlaceIntersectionUtil {
                 Asset2Data.Select(item => item.Key).ToSTR());
 
             if (Asset2Data.TryGetValue(intersectionInfo, out var assetData)) {
-                Log.Info("PlaceIntersectionUtil.ApplyTrafficRules(): assetData =" + assetData);
+                Log._Debug("PlaceIntersectionUtil.ApplyTrafficRules(): assetData =" + assetData);
             } else {
```
Comment (kianzarrin) [inline TLM/TLM/Util/PlaceIntersectionUtil.cs:58]: Just removing log spam here. no need to print this for all assets on load

File: TLM/TLM/Manager/Impl/LaneArrowManager.cs [hunk 2]
```diff
@@ -145,8 +145,8 @@ public class LaneArrowManager
         /// <param name="dedicatedTurningLanes">
         /// Narrow down updated roads to those that can have dedicated turning lanes.
         /// </param>
-        public void UpdateDedicatedTurningLanePolicy() {
-            Log.Info("UpdateAllDefaults(dedicatedTurningLanes:{dedicatedTurningLanes}) was called.");
+        public void UpdateDedicatedTurningLanePolicy(bool recalculateRoutings) {
+            Log.Info("UpdateDedicatedTurningLanePolicy() was called.");
```
[the remark this answers is not in the document]
Comment (kianzarrin) [reply TLM/TLM/Manager/Impl/LaneArrowManager.cs:149]: 👍 
yes.

File: TLM/TLM/Manager/Impl/LaneArrowManager.cs [hunk 3]
```diff
@@ -145,8 +145,8 @@ public class LaneArrowManager
         /// <param name="dedicatedTurningLanes">
```
[the remark this answers is not in the document]
Comment (aubergine10) [reply TLM/TLM/Manager/Impl/LaneArrowManager.cs:145]: `dedicatedTurningLanes` param does not exist

**Review by aubergine10 — APPROVED** [review]
LGTM 👍

**Review by kvakvs — APPROVED** [review]
Fine

**Pull request merged**